packages/peregrine/lib/talons/OrderHistoryPage/__tests__/__snapshots__/orderHistoryContext.spec.js.snap

```diff
@@ -4,7 +4,7 @@ exports[`renders children 1`] = `"Context Provider Children"`;
 
 exports[`returns default value for suffix 1`] = `
```

Comment:
The statement for this test could be updated, though I suppose empty string could be the default. I would just say `returns empty string if no suffix found in gql` or something like that.